### DISCUSSION TITLE
Qtile with Plasma

### DISCUSSION THREAD

**Discussion by ayghri** [qtile/qtile#4718] [Q&A] [answered]
I'm using i3+Plasma KDE combo and I'm exploring the same combination with Qtile.

I've managed to get things starting and I'm trying to solve the problem of the Plasma task bar overlapping with the tiled windows:
(Example here: Task bar covering GIMP menu)

![image](https://github.com/qtile/qtile/assets/32200675/f9c6d950-f71a-4191-89a1-95ebd489d979)

I've already got comments criticizing such choice, I'd appreciate some help

**Comment by elParaguayo** [comment, marked as answer]
If qtile isn't respecting the plasma bar then you can just add a `Gap` object to keep that part of the screen empty.

**Reply by ayghri** [reply]
I was hoping for a way to teach it some respect without having to resort to personal space tricks.

**Reply by elParaguayo** [reply]
Is the bar always visible or is it only visible when the mouse is at the top of the screen? If it's always visible then the gap "trick" is the easiest/quickest fix here.

**Reply by ayghri** [reply]
it's always visible.
I'll give it a try. Thank you
